packages/fxa-settings/src/components/LinkedAccounts/LinkedAccount.tsx

```diff
@@ -86,15 +86,15 @@ export function LinkedAccount({ providerId }: { providerId: number }) {
             </h2>
           </Localized>
 
-          <Localized id="la-unlink-content-2">
+          <Localized id="la-unlink-content-3">
             <p
               id="linked-accounts-unlink-description"
               className="my-4 text-center"
             >
               Are you sure you want to unlink your account? Unlinking your
-              account does not automatically sign you out of those services. To
-              do that you will need to manually sign out from the Connected
-              services section.
+              account does not automatically sign you out of your connected
+              services. To do that you will need to manually sign out from the
+              Connected services section.
```

Comment:
"Connected Services" is capitalized on the site so I feel like it should be capitalized here also.  What do you think?
![image](https://user-images.githubusercontent.com/80098/166582660-57f92102-efba-4a8a-9471-ecf82f76cac2.png)

Reply:
Oh I think that's a really good idea, I'll update it

Reply:
I think the second mention still needs capitalizing: "... from the Connected services section."